cmd/kk/pkg/plugins/network/modules.go

```diff
@@ -128,7 +129,8 @@ func deployCalico(d *DeployNetworkPluginModule) []task.Interface {
 			Template: templates.CalicoNew,
 			Dst:      filepath.Join(common.KubeConfigDir, templates.CalicoNew.Name()),
 			Data: util.Data{
-				"KubePodsCIDR":            d.KubeConf.Cluster.Network.KubePodsCIDR,
+				"KubePodsV4CIDR":            strings.Split(d.KubeConf.Cluster.Network.KubePodsCIDR, ",")[0],
+				"KubePodsV6CIDR":            strings.Split(d.KubeConf.Cluster.Network.KubePodsCIDR, ",")[1],
```

Comment:
Same as the InternalAddress, if the ipv6 address is not configured for the `KubePodsCIDR`, I think it should be reported `panic`.